homeassistant/components/sensor/gtfs.py

```diff
@@ -228,71 +301,201 @@ def unit_of_measurement(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return self._attributes
+        return {k: self._attributes[k] for k in sorted(self._attributes)}
```

Comment:
Revert this please. Also, this won't work in Python 3.5 as dictionaries are only ordered starting 3.6

Reply:
Backend should not do anything in the code to help the frontend. Frontend should do such things.

Reply:
Acknowledged, reverted!